.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Release
+on:
+  workflow_dispatch:
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+
+      - name: Print version
```

Comment:
"Print version" is a bit misleading, the purpose of this step is to make the RELEASE_VERSION available to subsequent steps. Maybe "Create release version name" or so would be better?